### ISSUE TITLE
M3 bolts can be used for servo

### ISSUE THREAD

**Issue by Akustav** [ut-robotics/picr21-team-bts#100]
The servos can be fastened by M3 bolts. Currently you plan to use what bolts?

![image](https://user-images.githubusercontent.com/5435954/151010576-6078ce45-ad7a-40ac-b39e-78eba9573722.png)

**Comment by Akustav** [comment]
The current way of fastening is impossible due to nuts not fitting: 
![image](https://user-images.githubusercontent.com/5435954/151030368-626097bd-2ffe-48b5-89d9-b8226b0dc991.png)

**Comment by mbz4** [comment]
than sand off 1 edge of the nut and fit it in place then thread it all in

**Comment by Akustav** [comment]
I think the current servo used in the model does not have exactly the same hole placement as the Corona servo without the rubber grommets. Makes it somewhat hard to judge the correct hole placement and clearance.

Overall grinding down one side of a nut is not a OK solution.

**Comment by mbz4** [comment]
in that case assume the "corona" servo is not the servo in use in this design
I'm reusing LoganPauls servo model unchanged

**Comment by Akustav** [comment]
But then the servo horns are different, since most of them are lost to time.

**Comment by mbz4** [comment]
added spacer 7.88mm :)

**Comment by mbz4** [comment]
if the tlo servo is the "correct" servo, then why didnt you simply provide the "correct" model
too late to cry now and can be disregarded 

# entirely

**Comment by Akustav** [comment]
I do not agree. Designing a advanced variant of the thrower requires components that are not part of the default kit. If you want to design an advanced variant with such components that we can provide then it is your responsibility to query us for the specifications of said components.

When checking chat history there has never been an inquiry about the exact make of servos that we can provide you. Sadly I can not read minds and need to work with more than one team during this course therefore initiative on the teams side in case they have questions or need further details is required.

I think it is not correct to disregard a design issue as important as servos you want to use.

**Comment by mbz4** [comment]
I queried for a parts list on 16 different occasions and it was the mentors sole responsibility in the end.

**Comment by mbz4** [comment]
The design for the servo was approved by someone last year. Therefore the design can WORK. Nowhere is it specified when its considered complete.
